travis/workspace.repos

```diff
@@ -3,3 +3,11 @@ repositories:
     type: git
     url: https://github.com/ros-controls/ros2_control.git
     version: master
+  control_msgs:
+    type: git
+    url: https://github.com/ros-controls/control_msgs.git
+    version: crystal-devel
```

Comment:
Is this intended? @bmagyar can we get a `dashing` and `foxy` branch?

Reply:
There are no dashing/foxy branches on https://github.com/ros-controls/control_msgs

Reply:
There were no changes made since the ROS2 port of `control_msgs`, hence why it is still on `crystal-devel`.